docs_user/modules/proc_migrating-existing-daemons-to-target-nodes.adoc

```diff
@@ -98,10 +93,9 @@ prometheus.cephstorage-1    cephstorage-1.redhat.local  172.17.3.53:9092
 ----
 +
 [NOTE]
-After you migrate the monitoring stack, you lose High Availability: the monitoring stack daemons have no VIP and HAproxy anymore; Node exporters are still running on all the nodes: instead of using labels we keep the current approach as we want to not reduce the monitoring space covered.
+After you migrate the monitoring stack, you lose high availability. The monitoring stack daemons no longer have a Virtual IP address and HAProxy anymore. Node exporters are still running on all the nodes. Instead of using labels we keep the current approach as we want to not reduce the monitoring space covered.
 
-. You must review the {Ceph}configuration to ensure that it is aligned with the relocation you just made. In particular, focus on the following configuration entries:
-//kgilliga: Note to self: Revisit the wording of this step.
+. Review the {Ceph} configuration to ensure that it aligns with the configuration on the target nodes. In particular, focus on the following configuration entries:
 +
```

Comment:
Please confirm that my rewrite of this step is accurate.

Reply:
correct, thank you!